### ISSUE TITLE
508-defect-2 [SCREENREADER]: On this page navigation should have a unique accessible label

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24998]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a screenreader user who uses landmarks to make sense of a page's structure, I want them to be uniquely labelled if there are any duplicate landmarks so I can tell the difference between them. For example, I'd like to read "On this page navigation" instead of "navigation" so I know it will jump me to the on this page navigation specifically and not the breadcrumb or footer.

## Details

If a page includes more than one navigation landmark, [each should have a unique label](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html). This helps folks with screen readers tell them apart when using a landmarks list.

## Acceptance Criteria

- [ ] On this page `nav` has `aria-labelledby="on-this-page"`

## Steps to Recreate

Using VoiceOver on Safari, open the rotor using `opt` + `cmd` + `u`
Navigate to landmarks
Confirm that there are two generic navigation landmarks

## Proposed Solution

Simply add `aria-labelledby="on-this-page"` to the "On this page" `nav` 

## WCAG or Vendor Guidance 

https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html

### Before

https://user-images.githubusercontent.com/14154792/118987179-0658c280-b94e-11eb-8e3f-ef9ce91c3c46.mov

### After

https://user-images.githubusercontent.com/14154792/118987199-0b1d7680-b94e-11eb-8745-994c11a985c6.mov

**Comment by mmiddaugh** [comment]
Will have @AngelaFowler82 validate this issue once we have a Vet Center published and available on Staging.

**Comment by mmiddaugh** [comment]
This can now be validated in Prod 🎉 
https://www.va.gov/nashville-vet-center/
https://www.va.gov/colorado-springs-vet-center/
@AngelaFowler82 @joshkimux

**Comment by AngelaFowler82** [comment]
All landmarks are uniquely labeled and their labels indicate their purpose. I think we can close this baby!
